lib/Service/SettingsService.php

```diff
@@ -149,18 +152,22 @@ public function getAll(string $uid, $saveInitial = false) : \stdClass {
 		$settings = $this->getSettingsFromDB($uid);
 		// use default for empty settings
 		$toBeSaved = false;
+		$toSaveNotesPath = false;
```

Comment:
I'm not a fan of the special handling here, can't we just call this method with saveInitial set to true in case of the capabilities?